Shared/Mobile-Rounds.ViewModels/Models/StationModel.cs

```diff
@@ -43,20 +44,16 @@ public class StationModel : NotificationBase
         public StationModel()
         {
             this.Items = new List<ItemModel>();
-            this.Navigate = new AsyncCommand((obj) =>
-            {
-                
-                BaseViewModel.Navigator.Navigate(Shared.Navigation.NavigationType.StationInput);
-            });
         }
-        public StationModel(string regionName, Guid stationId)
+
+        public StationModel(RegionModelSource region)
+            : this()
         {
-            this.Items = new List<ItemModel>();
             this.Navigate = new AsyncCommand(async(obj) =>
             {                
                 var file = Platform.ServiceResolver.Resolve<IFileHandler>();
                 var reads = await file.GetFileAsync("items.json");
-                var vm = new ReadingInputScreenViewModel(regionName, this.Name, reads, stationId);
+                var vm = new ReadingInputScreenViewModel(region, this, reads);
```

Comment:
@ChrisWillette See this change here? This is the **NEW** way of passing data to the new screen. By doing this, we can access this same `this.Navigate` command from the bread crumbs (see other comment).